model/training/data/image_lists/test.txt

```diff
@@ -0,0 +1,3356 @@
+PMC1242208_4.jpg
```

Comment:
Czy te zdjęcia z test i val są w katalogu images na google drive? I wtedy potrzebowalibyśmy oddzielnie tego formatu coco dla train, val i test zbiorów

Reply:
sprawdzę to